### PR TITLE
ISPN-6548 Removed warning thrown in WeldStandaloneTest

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-6548

### CODE REVIEW

File: integrationtests/cdi-weld-se-it/src/test/java/org/infinispan/integrationtests/cdi/weld/WeldStandaloneTest.java [hunk 1]
```diff
@@ -25,6 +25,9 @@ public void testWeldStandaloneInitialisation() throws Exception {
 
       //then
       assertEquals(retrievedValue, "abcd");
+
+      //cleanup
+      weld.shutdown();
```
Comment (danberindei) [inline integrationtests/cdi-weld-se-it/src/test/java/org/infinispan/integrationtests/cdi/weld/WeldStandaloneTest.java]: Shouldn't this be in a `finally` block?

Comment (slaskawi) [inline integrationtests/cdi-weld-se-it/src/test/java/org/infinispan/integrationtests/cdi/weld/WeldStandaloneTest.java]: In this particular test - no... Weld adds a shutdown hook. But let me add if for readability reasons

**Comment by pruivo** [comment]
pushing...

**Comment by pruivo** [comment]
integrated! thanks @slaskawi !

**Pull request closed without merging**